.gitignore

```diff
@@ -8,3 +8,4 @@
 !.circleci/**
 .idea/
 target
+bin
```

Comment:
```suggestion
```

Reply:
That's actually a good addition to .gitignore for eclipse IDEs